ci/ray_ci/automation/weekly_green_metric.py

```diff
@@ -34,5 +50,13 @@ def _get_release_blockers(
     return github.get_user().get_issues(state="open", labels=[release_blocker_label])
 
 
+def _get_oncall(issue: github.Issue.Issue) -> str:
+    labels = issue.get_labels()
+    for label in labels:
+        for team in TEAM:
+            if label.name == team:
+                return team
```

Comment:
this is basically `if label.name in TEAM: return label.name` ?

add unit test for this function / these changes?

Reply:
> this is basically `if label.name in TEAM: return label.name` ?

no?

Reply:
i moved the logic to `release/ray_release/test_automation/state_machine.py` so we can have some test cases